<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="p1progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p1end">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>player 1 end turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p2end">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>player 2 end turn</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="p2progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startgame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>start game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopgame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>stop game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gametime1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>120 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gametime2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5min</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameInfoText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>start new game via start button</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
